nexus/types/src/deployment/execution/mod.rs

```diff
@@ -3,7 +3,7 @@
 // file, You can obtain one at https://mozilla.org/MPL/2.0/.
 
 mod dns;
-mod overridables;
+pub mod overridables;
```

Comment:
Nit - there's already a `pub use overridables::*`, which means we're now exporting both `DEFAULT` and `overridables::DEFAULT` (plus dups of all the other `pub` things inside `overridables`). Some alternatives:

* Remove this pub and rename the static something like `OVERRIDABLES_DEFAULT`
* Make the static private and add a type-level method to get it (not sure what to name this; maybe `Overrideables::static_default() -> &'static Self`?)
* Remove the static and let callers use `Overridables::default()` directly

Reply:
Yeah... the problem is that the callers that provide their own value use the same value across multiple invocations, so I want `RealizeArgs` itself to store a reference.  But for cases where people don't care about this, there needs to be an owned value referring to the default _somewhere_.  That's what this static is for.  Using a method or `Overridables::default()` won't work.

I considered `OVERRIDABLES_DEFAULT` instead but was trying to be consistent with previous suggestions to use the module name when referring to the item instead of duplicating the module name in the item name.

Another approach is to _not_ `pub use overridables::*` here, but just expose `overridables::Overridables`.  I'm leaning towards that.

Reply:
> Another approach is to _not_ `pub use overridables::*` here, but just expose `overridables::Overridables`. I'm leaning towards that.

Seems fine :+1: 

> I considered OVERRIDABLES_DEFAULT instead but was trying to be consistent with previous suggestions to use the module name when referring to the item instead of duplicating the module name in the item name.

One more option, that I think I don't like but I'll at least mention it:

* keep the static as `DEFAULT`
* remove the `pub overridables` and the `pub use overridables::*`
* add `pub use overridables::Overridables` (to keep exporting that from this module)
* add `pub use overridables::DEFAULT as OVERRIDABLES_DEFAULT` (to respect the "don't duplicate module name" but also fix it up for pub exporting)